src/App.tsx

```diff
@@ -1,18 +1,49 @@
-import React from 'react';
-// import carsFromServer from './api/cars';
-// import colorsFromServer from './api/colors';
-
-// 1. Render car with color
-// 2. Add ability to filter car by brand name
-// 3. Add ability to filter car by color
+import React, { useState } from 'react';
+import carsFromServer from './api/cars';
+import colorsFromServer from './api/colors';
 
 export const App: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [chooseColor = colorsFromServer, setChooseColor] = useState('');
+  let displayedCars = carsFromServer
+    .filter(car => car.brand.toLowerCase()
+      .trim().includes(searchValue.toLowerCase()));
+
+  if (chooseColor) {
+    displayedCars = carsFromServer
+      .filter(car => car.brand.toLowerCase()
+        .trim().includes(searchValue.toLowerCase())
```

Comment:
What's the purpose of filtering by the name again? You can simplify it to avoid duplicating